daemon/cmd/app_logger.go

```diff
@@ -0,0 +1,145 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"go.uber.org/fx/fxevent"
+)
+
+type appLogger struct {
+	*logrus.Entry
+}
+
+func newAppLogger() fxevent.Logger {
+	return appLogger{Entry: log}
+}
+
+func (log appLogger) LogEvent(event fxevent.Event) {
```

Comment:
Why do we need to log all events? Is it helpful to have these printed while we are transition to uber/fx?